backend/setup.py

```diff
@@ -74,6 +74,10 @@
             "inspirehep_records = inspirehep.records:InspireRecords",
             "inspirehep_rt = inspirehep.rt:InspireRtExt",
         ],
+        "invenio_base.api_converters": [
+            "pid = inspirehep.pidstore.converters:DOIConverter",
+            "pidpath = inspirehep.pidstore.converters:DOIPathConverter",
```

Comment:
won't this kick-in for all PIDs, not only DOIs?

Reply:
From what i can understand we cannot assign more than one converter and path converter to pid/pidpath. So yes, it will affect all pids we use, but fortunately only doi's have letters.